src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/LayoutTests.kt

```diff
@@ -59,7 +72,8 @@ class LayoutTests
             } doReturn false
         }
 
-        val testModel = IndexViewModel(mockContext, listOf(), listOf(), listOf(), listOf(), true, false, null)
+        //val testModel = IndexViewModel(mockContext, listOf(), listOf(), listOf(), listOf(), true, false, null)
```

Comment:
Could you explain why this is commented out?

`mockContext` is now duplicated: in this function and in `testModel()`.

Reply:
That was meant for removal, I haven't re-requested review on this PR.